File: periodo-1/ip/atividade-2-beecrowd/idade-em-dias.c
#include <stdio.h>

int main() {
    int dias, meses, anos;

    scanf("%d", &dias);

    anos = dias / 365;
    meses = (dias % 365) / 30;
    dias = ((dias % 365) % 30);

    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);

    return 0;
}

File: periodo-1/ip/atividades/atividade-4-beecrowd/idade_em_dias.py
idade = int(input())

anos = int(idade / 365)
meses = int(idade % 365 / 30)
dias = int(idade % 365 % 30)

print(f"{anos} ano(s)")
print(f"{meses} mes(es)")
print(f"{dias} dia(s)")